modules/prebidServerBidAdapter/index.js

```diff
@@ -543,6 +545,17 @@ export const processPBSRequest = hook('sync', function (s2sBidRequest, bidReques
   }
 }, 'processPBSRequest');
 
+function emitNoBid(bid) {
+  const status = bid.nonbid.statuscode;
```

Comment:
should check that everything is defined 
 `bid && bid.nonbid && bid.nonbid.statuscode`

Reply:
This duplicates the check that happens when this function is called. Do we want a duplicate check?